main.go

```diff
@@ -351,8 +351,15 @@ func runGcloud(runner *Environ, workspace string, vargs GAE) error {
 		}
 	}
 
-	// add the app.yaml location
-	args = append(args, "./app.yaml")
+	switch {
+	// hook in the apropro yaml file
+	case vargs.DispatchFile != "":
+		args = append(args, "./dispatch.yaml")
+	case vargs.CronFile != "":
+		args = append(args, "./cron.yaml")
+	default:
+		args = append(args, "./app.yaml")
```

Comment:
It seems like these are mutually exclusive, is that true?   Could we just detect which file is present?

Reply:
@sbower all files may be present but the user can chose to use only some of them.

Reply:
I think folks could point the `app_file` config at a dispatch/cron file and the tool _would_ use it, but it may be a bit confusing since the other two configs already exist.

Reply:
I've updated some docs to clarify